src/components/Recipes/Recipes.tsx

```diff
@@ -30,7 +30,7 @@ const Recipes: React.FC = () => {
     <>
       <Card class='card'>
         <ul>
-          {foodprintCtx.recipes.items.map((item) => (
```

Comment:
Only show when it has 5 items?

Reply:
Forgot to take out when experimenting with different methods to prevent needing to put in the recipe list reset function, i'll take it out